src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <>
```

Comment:
do you really need this?

Reply:
Of course no)) movie list already have one main block...